src/test/chain-simulator/accounts.cs-e2e.ts

```diff
@@ -1187,4 +1187,270 @@ describe('Accounts e2e tests with chain simulator', () => {
+
+      for (const transfer of response.data) {
+        expect(transfer.sender).toBe(config.aliceAddress);
+      }
+    });
+
+    it('should return transfers with receiver parameter', async () => {
+      const receiverAddress = 'erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u';
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?receiver=${receiverAddress}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toBeGreaterThanOrEqual(1);
+
+      for (const transfer of response.data) {
+        expect(transfer.receiver).toBe(receiverAddress);
+      }
+    });
+
+    it('should return transfers with token parameter', async () => {
+      const accountTokens = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/tokens`);
+      const token = accountTokens.data[0].identifier;
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?token=${token}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toBeGreaterThanOrEqual(1);
+
+      for (const transfer of response.data) {
+        expect(transfer.function).toBe('ESDTTransfer');
+      }
+    });
+
+    it('should return transfers with senderShard parameter', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?senderShard=1`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toBeGreaterThanOrEqual(1);
+
+      for (const transfer of response.data) {
+        expect(transfer.senderShard).toBe(1);
+      }
+    });
+
+    it('should return transfers with receiverShard parameter', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?receiverShard=4294967295`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toBeGreaterThanOrEqual(1);
+
+      for (const transfer of response.data) {
+        expect(transfer.receiverShard).toBe(4294967295);
+      }
+    });
+
+    it('should return transfers with miniBlockHash parameter', async () => {
+      const miniBlockHashes = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?size=1`);
+      const miniBlockHash = miniBlockHashes.data[0].miniBlockHash;
+      const response = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?miniBlockHash=${miniBlockHash}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toStrictEqual(1);
+
+      for (const transfer of response.data) {
+        expect(transfer.miniBlockHash).toBe(miniBlockHash);
+      }
+    });
+
+    it('should return transfers with hashes parameter', async () => {
+      const hashes = await axios.get(`${config.apiServiceUrl}/accounts/${config.aliceAddress}/transfers?size=2`);
```

Comment:
create a global variable and use it.
At the beginning of the new test, you can define a local variable endpoint = "/.."

Reply:
no needed